pkg/envoy/lds/inmesh_test.go

```diff
@@ -625,3 +626,28 @@ func TestGetOutboundTCPFilter(t *testing.T) {
 		})
 	}
 }
+
+func TestGetOutboundHTTPFilter(t *testing.T) {
+	assert := tassert.New(t)
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	mockConfigurator := configurator.NewMockConfigurator(mockCtrl)
+	lb := &listenerBuilder{
+		cfg: mockConfigurator,
+	}
+
+	mockConfigurator.EXPECT().IsTracingEnabled().Return(false).AnyTimes()
+	mockConfigurator.EXPECT().GetTracingEndpoint().Return("test-endpoint").AnyTimes()
+	mockConfigurator.EXPECT().GetInboundExternalAuthConfig().Return(auth.ExtAuthConfig{
+		Enable: false,
+	}).AnyTimes()
+	mockConfigurator.EXPECT().GetFeatureFlags().Return(v1alpha1.FeatureFlags{
+		EnableWASMStats: false,
+	}).AnyTimes()
```

Comment:
Similar comment above about how if the values that these return aren't important, I'd remove them (though the `.AnyTimes()` is still necessary for the last two since they're called more than once):

This still passes:
```suggestion
	mockConfigurator.EXPECT().IsTracingEnabled()
	mockConfigurator.EXPECT().GetTracingEndpoint()
	mockConfigurator.EXPECT().GetInboundExternalAuthConfig().AnyTimes()
	mockConfigurator.EXPECT().GetFeatureFlags().AnyTimes()
```

(Acknowledging that this was just moved from a previous test and not something brand new in this PR)